awx/ui/client/lib/components/lookup-list/lookup-list.partial.html

```diff
@@ -0,0 +1,90 @@
+<div>
+    <div ng-hide="vm.collection.length === 0 && (searchTags | isEmpty)">
+        <div style="min-height: 20px"></div>
+        <smart-search
+            django-model="{{ vm.resourceName + 's' }}"
+            base-path="{{ vm.resourceName + 's' }}"
+            iterator="{{ vm.list.iterator }}"
+            dataset="vm.dataset"
+            list="vm.list"
+            collection="vm.collection"
+            default-params="vm.defaultParams"
+            query-set="vm.queryset"
+            search-tags="searchTags">
+        </smart-search>
+    </div>
+    <div
+        class="row"
+        ng-show="vm.collection.length === 0 && !(searchTags | isEmpty)"
+    >
+        <div class="col-lg-12 List-searchNoResults">
+            {{::vm.strings.get('NO_MATCH')}}
+        </div>
+    </div>
+    <div
+        class="List-noItems"
+        ng-show="vm.collection.length === 0 && (searchTags | isEmpty)"
+    >
+        {{::vm.strings.get('NO_ITEMS')}}
+    </div>
+    <div
+        class="list-table-container"
+        ng-show="vm.collection.length > 0"
+    >
+        <div
+            id="credential_input_source_table"
+            class="List-table"
+            is-extended="false"
+        >
+            <div class="List-lookupLayout List-tableHeaderRow">
+                <div></div>
+                <div class="d-flex h-100">
+                    <div
```

Comment:
this looks as if it is being passed a number of isolate scope attributes but it is a div (not an angular directive/component?)

Reply:
I agree with you. I generated the template using our state generator and list generator code as a starting point and then used it to create a somewhat portable component. My best guess is that something under the hood involving our legacy list styles, search, pagination, etc. is behind this, which makes me hesitant to change it right now since it works.

Reply:
We talked about this one - we're going to leave this as is.